pylintrc

```diff
@@ -7,7 +7,9 @@ disable=
  C0415, # Import outside toplevel
  E1101, # no-member
  R0904, # Too many public methods
+ R0903, # Too few public methods
```

Comment:
which class produces this ? does it make sense to block it on global level ? maybe we will put this into certain class ?

Reply:
In this case I prefer to put it here since we already have its complement (Too many public methods) above.  Also, both are what I consider "stupid" warnings by Pylint since it's common to create classes with too many and too few public methods.

Reply:
1. R0904 is here because when I was introducing pylint pipeline I have too many things to do so I needed to accept certain warnings to be able to merge whole idea . So as all other items R0904 is candidate to removal together with fix of the code which causing this error. 
2.  I can not agree that R0903 and R0904 are stupid . R0903 signaling you - "maybe your class can be replaced by dictionary ? "  and R0904 signaling you - "maybe you class will looks better as 2 separate classes ? ". Of course both questions may be ignored in some certain cases but I think blocking linter ability to ask them globally does not sounds good

Reply:
> 2. I can not agree that R0903 and R0904 are stupid . R0903 signaling you - "maybe your class can be replaced by dictionary ? "  and R0904 signaling you - "maybe you class will looks better as 2 separate classes ? ". Of course both questions may be ignored in some certain cases but I think blocking linter ability to ask them globally does not sounds good

The issue with Django is that subclassing with few methods is unavoidable, so I'm ignoring them at the class level now.

Removed both from pylintrc.  EC2 will need a refactor to decrease the number of methods from 23 to <= 20 but it's outside the scope of this PR.